### DISCUSSION TITLE
Re-enable ignored relay

### DISCUSSION THREAD

**Discussion by jornobe** [theyosh/TerrariumPI#708] [Q&A] [answered]
Is there a way to add ignored relays (Kasa in my case) back into the list? When trying to manually add it, logically, the system errors out because it is not a unique entry.

In the older version I would go to the config file and change properties there but now with the database behind I'm not sure where to start.

Tx!

**Comment by theyosh** [comment, marked as answer]
Hi,

the excluded devices can be revered at the 'Settings' page. There should be a list of excluded devices. Remove the relay from that list and save it.

And I think you need to restart TP4, so the excluded relay is loaded during startup.

**Reply by jornobe** [reply]
Thanks! Restart is indeed needed.
